x/fbridge/keeper/transfer_test.go

```diff
@@ -0,0 +1,62 @@
+package keeper
+
+import (
+	"encoding/binary"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	sdk "github.com/Finschia/finschia-sdk/types"
+	authtypes "github.com/Finschia/finschia-sdk/x/auth/types"
+	"github.com/Finschia/finschia-sdk/x/fbridge/testutil"
+	"github.com/Finschia/finschia-sdk/x/fbridge/types"
+)
+
+func TestHandleBridgeTransfer(t *testing.T) {
+	key, ctx, encCfg, authKeeper, bankKeeper := testutil.PrepareFbridgeTest(t)
+
+	sender := sdk.AccAddress("test")
+	amt := sdk.NewInt(1000000)
+	denom := "stake"
+	token := sdk.Coins{sdk.Coin{Denom: denom, Amount: amt}}
+
+	bridge := authtypes.NewEmptyModuleAccount("bridge")
+	authKeeper.EXPECT().GetModuleAddress(types.ModuleName).Return(bridge.GetAddress()).AnyTimes()
+	authKeeper.EXPECT().GetModuleAccount(ctx, types.ModuleName).Return(bridge).AnyTimes()
+	bankKeeper.EXPECT().SendCoinsFromAccountToModule(ctx, sender, types.ModuleName, token).Return(nil)
+	bankKeeper.EXPECT().BurnCoins(ctx, types.ModuleName, token).Return(nil)
+
+	k := NewKeeper(encCfg.Codec, key, authKeeper, bankKeeper, denom, "gov")
```

Comment:
The authority should be a valid address. You may introduce the corresponding verification into `NewKeeper()`.